db/src/main/kotlin/edu/artic/db/ApiRetrofit.kt

```diff
@@ -36,14 +38,26 @@ inline fun constructRetrofit(
         client: OkHttpClient,
         executor: Executor,
         moshi: Moshi,
+        xml: Boolean = false,
         schedulerFactory: CallAdapter.Factory = RxJava2CallAdapterFactory.createWithScheduler(Schedulers.io()),
-        customConfiguration: Retrofit.Builder.() -> Unit = {}): Retrofit =
```

Comment:
I think this should really've been done through the `customConfiguration` affordance. Did that not work out?

Reply:
Updated codebase to use `customConfiguration` for setting xml/json parsers.